Source/ACE.Database/Models/Shard/ShardDbContext.cs

```diff
@@ -39,12 +39,19 @@ public partial class ShardDbContext : DbContext
         public virtual DbSet<CharacterPropertiesShortcutBar> CharacterPropertiesShortcutBar { get; set; }
         public virtual DbSet<CharacterPropertiesSpellBar> CharacterPropertiesSpellBar { get; set; }
         public virtual DbSet<CharacterPropertiesTitleBook> CharacterPropertiesTitleBook { get; set; }
+        public virtual DbSet<ConfigPropertiesBoolean> ConfigPropertiesBoolean { get; set; }
+        public virtual DbSet<ConfigPropertiesDouble> ConfigPropertiesDouble { get; set; }
+        public virtual DbSet<ConfigPropertiesLong> ConfigPropertiesLong { get; set; }
+        public virtual DbSet<ConfigPropertiesString> ConfigPropertiesString { get; set; }
 
         protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
         {
-            var config = Common.ConfigManager.Config.MySql.Shard;
+            if (!optionsBuilder.IsConfigured)
```

Comment:
OnConfiguration will need to be manually reverted when you scaffold.

There should be no changes to this function as we add to the database.